conf/machine/juno.conf

```diff
@@ -0,0 +1,28 @@
+#@TYPE: Machine
+#@NAME: Juno vexpress
+#@DESCRIPTION: generic machine to be used by linaro-media-create
+
+require conf/machine/include/arm/arch-armv8.inc
+
+PREFERRED_PROVIDER_virtual/xserver ?= "xserver-xorg"
+
+XSERVER ?= "xserver-xorg \
+            mesa-driver-swrast \
+            xf86-input-evdev \
+            xf86-input-mouse \
+            xf86-video-fbdev \
+            xf86-input-keyboard"
+
+MACHINE_FEATURES = "usbhost usbgadget alsa screen wifi bluetooth efi optee"
+
+PREFERRED_PROVIDER_virtual/kernel ?= "linux-armlt"
+
+KERNEL_IMAGETYPE = "Image"
+KERNEL_DEVICETREE = "arm/juno.dtb arm/juno-r1.dtb arm/juno-r2.dtb"
+
+OPTEEMACHINE = "vexpress-juno"
```

Comment:
why do we use OPTEEMACHINE? For HiKey, we use UEFIMACHINE.

It seems we introduce new variables and we don't use them later on (or at least I don't see where we use them).